datastore/dbstore.go/communicationdbmethod.go

```diff
@@ -1,4 +1,6 @@
-package database
+package dbstore
+
+var communicationDbMethod CommunicationDatabaseMethod
```

Comment:
Why initialize here? Wouldn't it be better to do it where it being used?

Reply:
I don't know.  I guess I figured it's all defined in that file, so why not put it in that file.

It's in the same scope either way.
Which is better?

Reply:
I think it’s better to put it where it’s calling those methods. The original thought behind db method was to establish all the methods needed to query the db. Then, in the main file where those methods are being used, create a instance.